draft-ietf-mls-architecture.md

```diff
@@ -1019,13 +1019,13 @@ interoperate.
 
 - Additional services may or may not be required depending on the application
   design:
-  - If assisted joining is desired (meaning that the ratchet tree is not
+  - If assisted joining is desired (meaning that the group state is not
     provided in Welcome messages), there must be a method to download the
-    ratchet tree corresponding to a group.
+    group state corresponding to a group.
   - If assisted joining is desired and the Delivery Service is not able to
-    compute the ratchet tree itself (because some proposals or commits are sent
+    compute the group state itself (because some proposals or commits are sent
```

Comment:
```suggestion
    compute the group state itself, including the corresponding `ratchet_tree`
    object, (for example, because some proposals or commits are sent
```